tests/scripts/thread-cert/node.py

```diff
@@ -940,11 +940,11 @@ def set_parent_priority(self, priority):
         self._expect('Done')
 
     def get_partition_id(self):
-        self.send_command('leaderpartitionid')
+        self.send_command('leaderpartitionid preferred')
```

Comment:
keep it as is, and add get_preferred_partition_id() method?

Reply:
Updated